### DISCUSSION TITLE
Block moving model when using UI elements

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#3476] [Q&A] [answered]
Hi guys, in my AR session i have a model and one UI element (its a small slider that toggles an on/off state for the model). Whenever i press the slider if the model is behind it i slightly rotate it or move it a little, is there a way to cancel out interaction with the model while interacting with the UI?

**Comment by samaneh-kazemi** [comment]
Hi @danilokrulj, thanks for reaching out. Can you please share a glitch link so we can debug easier? Also, providing a more details such as browser/phone and repro steps and video would be helpful. 
You can remix this glitch link: https://glitch.com/edit/#!/model-viewer

**Comment by elalish** [comment, marked as answer]
I believe you're looking for this:
```
document.querySelector("#your-element").addEventListener('beforexrselect', (ev) => {
    // Keep slider interactions from affecting the XR scene.
    ev.preventDefault();
  });
```
From here: https://modelviewer.dev/examples/augmentedreality/#webXR

**Reply by danilokrulj** [reply]
Forgot to say thank you :) thank you for your response and your time
